### PR TITLE
Feature: Create a toggle to show or hide the global styles doc [ED-9577]

### PR DESCRIPTION
**Pull request by mark-elementor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/kits/assets/js/hooks/ui/document/save/save/after.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export class KitAfterSave extends After {
 
 		if ( elementor.activeBreakpointsUpdated ) {
 			const reloadConfirm = elementorCommon.dialogsManager.createWidget( 'alert', {
-				id: 'elementor-save-kit-refersh-page',
+				id: 'elementor-save-kit-refresh-page',
```
Comment (ghost) [inline core/kits/assets/js/hooks/ui/document/save/save/after.js:40]: Are you sure it's safe to change it?
No one using this ID?

File: core/kits/documents/tabs/global-colors.php [hunk 2]
```diff
@@ -45,6 +55,21 @@ protected function register_tab_controls() {
...
 
+		$this->add_control(
+			'colors_enable_style_guide_preview',
+			[
+				'label' => esc_html__( 'Style Guide Preview', 'elementor' ),
+				'type' => Global_Style_Switcher::CONTROL_TYPE,
+				'description' => esc_html__( 'Switch between the content area and style guide to preview your changes to global colors.', 'elementor' ),
+				'separator' => 'after',
+				'label_off' => esc_html__( 'Off', 'elementor-pro' ),
+				'label_on' => esc_html__( 'On', 'elementor-pro' ),
```
Comment (ghost) [inline core/kits/documents/tabs/global-colors.php]: Fix domain

File: core/kits/documents/tabs/global-typography.php [hunk 3]
```diff
@@ -49,6 +59,21 @@ protected function register_tab_controls() {
...
 
+		$this->add_control(
+			'typography_enable_style_guide_preview',
+			[
+				'label' => esc_html__( 'Style Guide Preview', 'elementor' ),
+				'type' => Global_Style_Switcher::CONTROL_TYPE,
+				'description' => esc_html__( 'Switch between the content area and style guide to preview your changes to global fonts.', 'elementor' ),
+				'separator' => 'after',
+				'label_off' => esc_html__( 'Off', 'elementor-pro' ),
+				'label_on' => esc_html__( 'On', 'elementor-pro' ),
```
Comment (ghost) [inline core/kits/documents/tabs/global-typography.php]: fix domain

File: core/kits/documents/tabs/global-colors.php [hunk 4]
```diff
@@ -36,6 +38,14 @@ public function get_help_url() {
 		return 'https://go.elementor.com/global-colors/';
 	}
 
+	public function get_action_show() {
+		return 'elementor/preview/style-guide/colors';
```
Comment (ghost) [inline core/kits/documents/tabs/global-colors.php]: "style-guide"
"design-guidelines"
"global-style"
"styleguide"

Please choose ONE

**Review by ghost — CHANGES_REQUESTED** [review]

File: core/kits/documents/tabs/global-colors.php (hunk 2 above)
Comment (mark-elementor) [reply core/kits/documents/tabs/global-colors.php]: Fixed.

File: core/kits/documents/tabs/global-typography.php (hunk 3 above)
Comment (mark-elementor) [reply core/kits/documents/tabs/global-typography.php]: Fixed.

File: core/kits/assets/js/hooks/ui/document/save/save/after.js (hunk 1 above)
Comment (mark-elementor) [reply core/kits/assets/js/hooks/ui/document/save/save/after.js:40]: I didn't find any uses. Feel quite confident to change.

File: core/kits/documents/tabs/global-colors.php (hunk 4 above)
Comment (mark-elementor) [reply core/kits/documents/tabs/global-colors.php]: Updated everywhere possible.

**Pull request merged**